src/docs/product/integrations/index.mdx

```diff
@@ -8,6 +8,10 @@ redirect_from:
   - /workflow/integrations/global-integrations/
   - /workflow/legacy-integrations/
   - /workflow/global-integrations/
+  - /integrations/azure-devops/?utm_source=blog&utm_m&code=swift
+  - /product/integrations/notification-incidents/discord/
```

Comment:
```suggestion
  - /product/integrations/notification-incidents/discord/
```
This should be redirected here: https://docs.sentry.io/product/accounts/early-adopter-features/discord/?utm_m=null&utm_source=blog